docs/update_docs.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
 pushd ${0%/*}
-bazel build //... && cp bazel-bin/*.md . && chmod 0644 *.md && git add *.md && git commit -m "Regenerate documentation"
+bazel clean --expunge && bazel build //... && cp bazel-bin/*.md . && chmod 0644 *.md && git add *.md && git commit -m "Regenerate documentation"
```

Comment:
Why do you want to do a clean expunge here?

Reply:
I ran into a case where because I generated docs for my `cbindgen` branch and never ran `clean` here, I kept having the `cbindgen` docs appear in branches where it shouldn't. The `--expunge` was just me being overly aggressive in avoiding this issue. I feel strongly that `clean` should be here but can remove `--expunge`. Thoughts?

Reply:
Right now I understand, without --expunge should be enough. Add a comment that we clean to remove stalled artifact

Reply:
Done